File: laij84/w04d03/lib/shelter.rb
class Shelter

  attr_reader :name, :clients

  def initialize(name)
    @name = name
    @clients = []
  end

  def add_client(name, age, gender, number_of_pets)
    client = Client.new(name, age, gender, number_of_pets)
    @clients << client
  end

end

File: laij84/w08d01/starter-code 6/models/shelter.js
// require mongoose
var mongoose = require("mongoose");

// require the Animal and Client models
var Animal = require("./animal");
var Client = require("./client");

// create the shelterSchema
var shelterSchema = new mongoose.Schema({
  name: String,
  animals: [Animal.schema],
  clients: [Client.schema]
});

// export the model
module.exports = mongoose.model("Shelter", shelterSchema);






